### ISSUE TITLE
test in Platius model 

### ISSUE THREAD

**Issue by trinh-hoang-hiep** [cnguyen10/few_shot_meta_learning#18]
Hi Cuong,
Thanks for your great work.
I want to ask about the test function in Platius.py what is the meaning of eps_generator in test function

**Comment by cnguyen10** [comment]
Hi Hiep,

The code had been developed a couple of years ago. I then decided to rewrite to put it into one structure specified in `MLBase.py`. Thus, there might remain code adopted from previous implementation.

As what I can recall, the prefix `eps_` denotes "episode" or "task" in meta-learning. Thus, the `eps_generator` is an interative object/function that generates the training and validation split of a task. This had been done before implementing the `EpisodeSampler` class in `EpisodeSampler.py`. In that implementation, a task was generated by simply sampling a number of class labels, e.g. sampling 5 class labels out of 64 classes in mini-ImageNet followed by sampling training and validation split images based on those 5 classes. Such allowed us to gain more control over the tasks/episodes generated rather than a random one. With the implementation of `EpisodeSampler` class, the task generation was integrated into PyTorch dataloader structure, and can run much faster than the old implementation relying on manually sampling tasks.

By the way, if you just want to discuss about the meaning of the code, please open a discussion instead of issue.

**Comment by trinh-hoang-hiep** [comment]
Thanks for your detailed explanation!
a bit off-topic question, I want to apply Platius or vampire to the uncertainty in my image segmentation task. My task is a single task, so there will be no validation sample x_v and training sample x_t, so is there any way I can apply this model to a single task?
